spec/models/your_details_form_spec.rb

```diff
@@ -1,7 +1,36 @@
 require 'spec_helper'
 
 describe Wpcc::YourDetailsForm, type: :model do
-  subject { described_class.new(age: 32, gender: 'female', salary: 35_000) }
+  let(:salary)                  { 35_000 }
+  let(:salary_frequency)        { 'year' }
+  let(:contribution_preference) { 'full' }
+
+  describe '.load_config' do
+    context 'file exists' do
+      let(:salary_threshold_config_file) do
+        Wpcc::Engine.root.join('config', 'salary_threshold.yml')
+      end
+
+      specify { expect(salary_threshold_config_file).to exist }
+    end
+
+    it 'returns salary frequencies' do
+      expect(Wpcc::YourDetailsForm::SALARY_FREQUENCIES).to eq(
+        %w[year
+           month
```

Comment:
@willhall88 they won't fit on one line that's why I had to split them unto multiple lines.